docs/content/docs/_index.md

```diff
@@ -1086,6 +1086,12 @@ Example: `{{ sections | get(key="posts/content") }}`
 Split a string into an array of strings, separated by a pattern given.
 Example: `{{ path | split(pat="/") }}`
 
+#### int
+Converts a value into an integer.  The `default` argument can be used to specify the value to return on error, and the `base` argument can be used to specify how to interpret the number.  Bases of 2, 8, and 16 understand the prefix 0b, 0o, 0x, respectively.
```

Comment:
I would prefer starting by not having the bases, it seems extremely niche and I can't even think of a usecase for it